datacenter/dtr/2.2/guides/admin/configure/job-runner.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Jobrunner
+description: Learn about the inner-workings of the jobrunner container in the DTR workflow.
+keywords: docker, job, runner
+---
+
+The jobrunner is presented as a container in DTR. It is a DTR mechanism that:
+
+1. Consumes jobs from a universal jobs queue 
```

Comment:
I would use `cluster-wide` as opposed to `universal` to be more specific about the scope of the queue.